src/cornerstone-core.js

```diff
@@ -0,0 +1 @@
+export * from '../../cornerstone/src/index.js';
```

Comment:
I don't think I can merge this. Won't it break users that are using module systems? (e.g. import * as cornerstone from 'cornerstone-core'?